README.md

```diff
@@ -73,4 +87,23 @@ After you create a new repository using this repo as template, please follow the
     11.3: help/
 
 
-11. Clear the content of this file and add your own README.md as the project README file. We recommend putting badges of the project status at the top of the README file. For more information about badges, please refer to [shields.io](https://shields.io/).
+11. Clear the content of this file and add your own README.md as the project README file. We recommend putting
+badges of the project status at the top of the README file. For more information about badges, please refer
+to [shields.io](https://shields.io/).
+
+## Repository Adjustments
+
+### Add an access token to anaconda
+Here we assume your intent is to
+upload the conda package to the [anaconda.org/neutrons](https://anaconda.org/neutrons) organization.
+An administrator of _anaconda.org/neutrons_ must create an access token for your repository
+in the [access settings](https://anaconda.org/neutrons/settings/access).
+
+After created, the token must be stored in a _repository secret_:
+1. Navigate to the main page of the repository on GitHub.com.
+2. Click on the "Settings" tab.
+3. In the left sidebar, navigate to the "Security" section and select "Secrets and variables" followed by "Actions".
+4. Click on the "New repository secret" button.
+5. Enter `ANACONDA_TOKEN` for the secret name
+6. Paste the Anaconda access token
+7. Click on the "Add secret" button
```

Comment:
Can you add an item 8 about testing by creating a release candidate? I don't think we are worried about lots of extra release candidates showing up

Reply:
done